src/Confluent.Kafka/PartitionMetadata.cs

```diff
@@ -27,6 +27,24 @@ namespace Confluent.Kafka
     /// </summary>
     public class PartitionMetadata
     {
+        /// <summary>
```

Comment:
Minor improvement, but good point - it would be better if they weren't. I'll make an issue for this (should be a separate PR). It's an API change, but we can probably get away with just making the changes as the metadata methods are marked as unstable (even if these classes aren't).